api/graphql/makeModels.js

```diff
@@ -273,11 +274,9 @@ export default async function makeModels (userId, isAdmin) {
         {followers: {alias: 'participants'}},
         {comments: {alias: 'messages', typename: 'Message', querySet: true}}
       ],
-      filter: nonAdminFilter(relation => relation.query(q => {
+      filter: relation => relation.query(q =>
```

Comment:
Why remove `nonAdminFilter` here?

Reply:
Basically because there's no need for you to see other people's message threads even if you're an admin.